presets/4.3/tune/PabloFlor_Freestyle.txt

```diff
@@ -0,0 +1,32 @@
+#$ TITLE: Pablo Flor/Fenix Fpv Tune
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: youtube, Fenix, freestyle, Pablo Flor, HD
+#$ AUTHOR: Pablo Flor
+#$ DESCRIPTION: Pablo Flor / Fenix Fpv Pid´s for 5inch Freestyle builds
+#$ DESCRIPTION:
+#$ DESCRIPTION: This configuartion is intended for freestyle quads, specially for setups around 730g AUW (with Gopro and Lipo). It works the best for 6s quads with 1800-1900kv motors and medium pitch props (5143). Nevertheless I have tested this tune on other 4s quads and the result is great!! 
+#$ DESCRIPTION: if any high throttle vibration is noticed, lowering the master multiplier slider should solve it.
+#$ DESCRIPTION: 
+#$ DESCRIPTION: I have worked on this tune to minimize propwash and make the quad feel "on rails" which really helps for hitting gaps and general precission flying. It also makes for a very responsive flight characteristics which is perferct for agressive freestyle eventhough my main focus is smooth and flowy flights.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Note: I also use the RPM Filtering for clean builds preset, so as a recommendation, load also that one if your quad supports it.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Enjoy!
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+
+#  -- PID settings -- 
+
+set anti_gravity_gain = 4000
+set feedforward_jitter_factor = 12
+set simplified_master_multiplier = 120
+set simplified_i_gain = 75
+set simplified_d_gain = 115
+set simplified_pi_gain = 115
+set simplified_dmax_gain = 0
+set simplified_feedforward_gain = 95
+set simplified_pitch_d_gain = 115
+set simplified_dterm_filter = OFF
+set simplified_dterm_filter_multiplier = 0
+
```

Comment:
because you are using sliders tune you have to add
"simplified_tuning apply" command at the end of the preset.

Otherwise, CLI won't recalculate the actual PID values, so you will have default PIDs and changed sliders.